interface-elements-for-web/articles/menu/responsive-layout.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Responsive Layout
-author: Anna Kondratova
+author: Svetlana Nikulina
 ---
 # Responsive Layout
 
```

Comment:
"The menu can operate in the following ways when items do not fit the browser window's width" Why 'operate'? Are these options you need to specify?

Reply:
"The menu moves its items to a popup menu hidden under the ellipsis button" -> The menu moves its items to a drop-down menu. To show these items, hover the ellipsis button.

Reply:
"The menu hides root items' text and displays icons only. Hover an item to display its text" -> The menu displays icons only. When you hover an item, its text is displayed as a tooltip.

Reply:
Can I say "When menu items do not fit the browser window's width, the menu can behave as follows:" instead? Or act/work/function is better?

Reply:
Perhaps: The menu can display its items as follows when they do not fit the browser window's width:

Reply:
- Move items to a drop-down menu. To show these items, hover the ellipsis button.
- Display icons only. When you hover an item, its text is displayed as a tooltip.
- Orientate vertically and hide items under the hamburger button. You can click this button to show vertical side-menu.